@app/server/src/plugins/PassportLoginPlugin.ts

```diff
@@ -191,29 +208,15 @@ const PassportLoginPlugin = makeExtendSchemaPlugin((build) => {
             await login({ session_id: session.uuid });
           }
 
-          // Get session_id from PG
-          await pgClient.query(
-            `select set_config('jwt.claims.session_id', $1, true)`,
-            [session.uuid]
-          );
+          // Update pgSettings so future queries will use the new session
+          pgSettings!["jwt.claims.session_id"] = session.uuid;
 
-          // Fetch the data that was requested from GraphQL, and return it
-          const sql = build.pgSql;
-          const [row] = await selectGraphQLResultFromTable(
-            sql.fragment`app_public.users`,
-            (tableAlias, sqlBuilder) => {
-              sqlBuilder.where(
-                sql.fragment`${tableAlias}.id = app_public.current_user_id()`
-              );
-            }
-          );
-          return {
-            data: row,
-          };
+          return {};
```

Comment:
It's now the responsibility of the next layer of plans to execute this. See the `LoginPayload.user` plan above, which is effectively:

```ts
    LoginPayload: {
      user() {
        const $userId = currentUserIdSource.execute();
        return userSource.get({ id: $userId });
      },
    }
```